packages/spotlight/src/utils.js

```diff
@@ -172,13 +172,47 @@ function getContainerRect (containerId) {
 	return getRect(containerNode);
 }
 
+function isStandardFocusable (element) {
+	if ((element.tabIndex < 0) || isAtagWithoutHref(element) || isActuallyDisabled(element) || isExpresslyInert(element) || isElementHidden(element)) {
```

Comment:
For later visits, could you please add some comments for clauses in this function?

Reply:
I added some comments